thats_the_way.py

```diff
@@ -0,0 +1,17 @@
+from os import listdir
+from os.path import isfile, join
+
+
+def that_the_way(path):
```

Comment:
and also, its "that's the way"

Reply:
Hey Tal,
I knew I should have written "that's" and not "that", but I can't write the sign ' in the function's name
and Pycharm has put a green line under the word and I couldn't live with that.

I have read all your messages and I am embarrassed to see so many things I have to fix.
I will fix them ASAP.

Thank you!